protocol/flatbuffers/settings/model.fbs

```diff
@@ -0,0 +1,29 @@
+namespace solarxr_protocol.rpc.settings;
+
+/// Settings for the skeletal model that are toggles.
+table ModelToggles{
+    extended_spine: bool;
```

Comment:
reminder to self: check to ensure these are optional, so that I don't need to specify all settings when I modify settings

Reply:
Oh ya, probably important to do that....

Reply:
Same with the ratios btw!